src/daemon.js

```diff
@@ -12,13 +12,13 @@ const DB = require('./dbs'),
       fatality = require('./fatality');
 
 const newDeployment = deployDoc =>
-  !!deployDoc &&
-  deployDoc._id === HORTI_UPGRADE_DOC &&
```

Comment:
I'm not totally sure about the change here. I'm unsure why `newDeployment` is confirming the _id of the deployment document.  I'm hoping it is an unnecesssary check, but would like thoughts.